backend/Dockerfile

```diff
@@ -27,6 +27,13 @@ RUN mvn dependency:go-offline
 COPY src/ /app/src/
 RUN mvn clean package -DskipTests
 
+# Copy Legal information for distributions
+COPY README.md /opt/app/README.md
```

Comment:
Target state is to have the following:
- workflow that copies the files to this directory and includes the files in distribution
- local docker build that ignores the files

As a result we only want to maintain the files on root level.

When building locally, this fails.

We could do this as follows with a one-liner: 
``` Dockerfile
# Copy Legal information for distributions
COPY DOCKER_NOTICE.md README.md SECURITY.md* LICENSE* DEPENDENCIES* /opt/app/
```
It works as the first two files do always exist (btw we incorrectly had the wrong notice in use). Using the wildcard afterwards allows the conditional copy (there is no arg-driven conditional copy in docker).

Please also remove the other files: NOTICE.md, SECURITY.md, DEPENDENCIES. Do the same for the frontend (remove files, update dockerfile).
Within the docs/DEVELOPMENT.md, please update the commands (no copy to frontend/backend) needed anymore